projects/rero/ng-core/src/lib/record/editor/widgets/load-template-form/load-template-form.component.html

```diff
@@ -0,0 +1,39 @@
+<!--
+  RERO ILS UI
```

Comment:
ng-core license? Can you check for other files like this?

Reply:
done. I checked other files and found another RERO-UI annotation.